ls1intum/Artemis
# `Dark Mode`: Change used emoji-set to avoid hardly noticeable emojis

### Checklist
#### General


#### Client

### Motivation and Context
Solves #5118 .

### Description
As reported by @willergeorg, some emojis (from the group 'heavy...') are hardly noticeable within the emoji picker, Posts and Answers when using the darkmode. Since the problem could also be replicated via the used libraries own demo page, we change the used emoji-set to 'facebook', where the emojis are noticeable again.

### Steps for Testing

Prerequisites:
- 1 Students
- 1 Lecture with Discussions enabled

1. Log in to Artemis
2. Navigate to Course Discussions page or an exercise
3. Switch to dark mode
4. Create a Post if none exists.
5. Open the emoji-picker and type 'heavy' to it's searchbar.
6. Make sure that the emojis are more noticeable then they were before.
7. Upvote a post by selecting the 'plus' emoji.
8. Make sure that the upvote is visible.
9. Make sure the same of the steps holds also for AnswerPosts.

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage


Unchanged

### Screenshots


Old emoji-set
![image](https://user-images.githubusercontent.com/67344158/169050837-117cee51-15a8-44db-b280-c06311e108c2.png)!
New emoji-set
![image](https://user-images.githubusercontent.com/67344158/169050704-139b4d97-489c-4f5f-9cbe-7b1056c76768.png)!
